src/tests/testIpAddress.cc

```diff
@@ -8,6 +8,8 @@
 
 #include "squid.h"
 #include "compat/cppunit.h"
+#include "compat/netdb.h"
```

Comment:
With this please remove the redundant `netdb.h` include below.

Reply:
removed